services/api/tests/src/test_admin_org.py

```diff
@@ -268,8 +268,10 @@ def test_modify_org_sql_exception(mock_pgquery, mock_check_safe_input):
 
 
 @patch("api.src.admin_org.pgquery.write_db")
-def test_delete_org(mock_write_db):
-    expected_result = {"message": "Organization successfully deleted"}
+@patch("api.src.admin_org.pgquery.query_db")
+def test_delete_org(mock_query_db, mock_write_db):
```

Comment:
Do we have a corresponding test for the fail case here?

Reply:
Added two new tests - one for the failure case with an orphan RSU and one for the failure case with an orphan user